doc/advanced_topics.rst

```diff
@@ -0,0 +1,331 @@
+###############
+Advanced Topics
+###############
+
+In this section, we illustrate some topics which experienced
```

Comment:
Depending on whether we go with "advanced topics" or not, we would want to reword to remove "experience" as a prerequisite for looking at the model and script sections.

Reply:
OK, rephrased